site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -875,6 +901,128 @@ This strategy instructions FusionAuth when to append a unique suffix to the user
 * `Always` - Always append a unique suffix even when the requested username is not in use.
 * `OnCollision` - Only append a unique suffix when the requested username is in use.
 
+[field]#tenant.webAuthnConfiguration.bootstrapWorkflow.authenticatorAttachmentPreference# [type]#[String]# [optional]#Optional# [default]#defaults to `crossPlatform`# [since]#Available since 1.41.0#::
+Determines the authenticator attachment requirement for WebAuthn passkey registration when using the bootstrap workflow. The possible values are:
++
+* `any` - An authenticator with any attachment modality will be allowed during registration.
+* `crossPlatform` - Only authenticators with the `cross-platform` attachment modality will be allowed during registration. These are also referred to as "roaming" authenticators.
+* `platform` - Only authenticators with the `platform` attachment modality will be allowed during registration.
+
++
+The recommended value for the bootstrap workflow is `crossPlatform` (when supported).
+
++
+:premium_feature: WebAuthn
+include::../shared/_premium-edition-blurb-api.adoc[]
+// You must keep the empty line above the `+` and you cannot undefine :premium_feature!: here it breaks the formatting because of the list.
+
++
+:enterprise_feature: WebAuthn roaming authenticators
+include::../shared/_enterprise-edition-blurb-api.adoc[]
+// You must keep the empty line above the `+` and you cannot undefine :enterprise_feature!: here it breaks the formatting because of the list.
+
+[field]#tenant.webAuthnConfiguration.bootstrapWorkflow.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.41.0#::
+Whether or not this tenant has the WebAuthn bootstrap workflow enabled. The bootstrap workflow is used to perform a manual login using WebAuthn and can be used on a new device.
++
+:premium_feature: WebAuthn
+include::../shared/_premium-edition-blurb-api.adoc[]
++
+:premium_feature!:
+
+[field]#tenant.webAuthnConfiguration.bootstrapWorkflow.userVerificationRequirement# [type]#[String]# [optional]#Optional# [default]#defaults to `required`# [since]#Available since 1.41.0#::
```

Comment:
Should all of these fields have something like "Required if `tenant.webAuthnConfiguration.bootstrapWorkflow.enabled` is `true`."?

I've seen something similar for other items here. I think that all of the WebAuthn stuff will still work with defaults.

Reply:
These will still default if they are omitted from the request object, so we don't need the message.